md2html.sh

```diff
@@ -1,27 +1,52 @@
 #!/bin/bash
 
 set -euxo pipefail
+
 convert_link=0
+
 while getopts "hl" opt; do
-	case "$opt" in
-		h)
-			echo "Usage: $0 [-h] [[-l] md_file0 md_file1 ...]"
-			exit ;;
-		l)
-			convert_link=1
-			shift ;;
-		*)
-			;;
-	esac
+  case "$opt" in
+    h)
+      echo "Usage: $0 [-h] [[-l] md_file0 md_file1 ...]"
+      echo "options:"
+      echo "-l  convert links to other md files to html"
+      exit ;;
+    l)
+      convert_link=1
+      shift ;;
+    *)
+      ;;
+  esac
 done
 
 for file in "$@"; do
-	pandoc -f markdown_github "${file}" -t asciidoc -o "${file%.md}.tmp.adoc"
-	if [ "$convert_link" -ne 0 ]; then
-		sed -i -e "s/\.md\>/\.html/g" "${file%.md}.tmp.adoc"
-	fi
-	touch -r "${file}" "${file%.md}.tmp.adoc"
-	TZ=UTC asciidoc -o "${file%.md}.html" -a footer-style=none -a toc2 -a source-highlighter=highlight "${file%.md}.tmp.adoc"
-	tr -d '\r' < "${file%.md}.html" > "${file%.md}.tmp.adoc"
-	mv "${file%.md}.tmp.adoc" "${file%.md}.html"
+  md2html_tool=""
+  # RHEL 9 in brew cannot use pandoc, hence trying kramown first
+  if type -p kramdown >/dev/null && kramdown -i GFM < /dev/null > /dev/null; then
+    md2html_tool=kramdown
+  elif type -p pandoc >/dev/null; then
+    md2html_tool=pandoc
+  fi
+  # With kramdown, convert directly to HTML
+  if [ $md2html_tool == kramdown ]; then
```

Comment:
use double quotes around `$md2html_tool` since its value could be `""` and then the `if` will evaulate to this:
```
if [ == kramdown ]
```
which will throw a strange error

Reply:
Thanks, resolved this